nav2_planner/src/planner_server.cpp

```diff
@@ -336,11 +336,13 @@ void PlannerServer::computePlanThroughPoses()
 
   auto start_time = steady_clock_.now();
 
-  // Initialize the ComputePathToPose goal and result
+  // Initialize the ComputePathThroughPoses goal and result
   auto goal = action_server_poses_->get_current_goal();
   auto result = std::make_shared<ActionThroughPoses::Result>();
   nav_msgs::msg::Path concat_path;
 
+  geometry_msgs::msg::PoseStamped curr_start, curr_goal;
```

Comment:
Move back to original location near where its first use is

Reply:
The curr_start and goal need to be outside of the try catch in order to print out the offending start/goal if an exception is thrown.